google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/TopicSnippets.java

```diff
@@ -114,94 +114,6 @@ public boolean deleteAsync() throws ExecutionException, InterruptedException {
   }
 
   /**
-   * Example of publishing one message to the topic.
-   */
-  // [TARGET publish(Message)]
-  public String publishOneMessage() {
```

Comment:
Yes. This is related to what you pointed out above. I'm still not sure how to expose the options. So, if we write the samples now, I think we'll be duplicating work if we change our minds about how to expose them. These open issues are tracked here: https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1493